app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -84,3 +91,47 @@ def get_all_records():
         }
         send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
         return request_info
+
+def ingest_sample_frame_goi_filter_options():
+    url = "https://api.airtable.com/v0/appFfYIupTrVQ0HJx/Sample%20Frame%20GOI?fields=Order&fields=Name&sort%5B0%5D%5Bfield%5D=Order&fields=French%20Name"
+    headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+
+    # Make request and retrieve records in json format
+    r = requests.get(url, headers=headers)
+    data = r.json()
+
+    # Try to get records from data if the request was successful
+    try:
+        # Get sorted records
+        records = data["records"]
+        drop_specific_table_entries(datetime.now(), PopulationGroupOptions)
```

Comment:
Won't this just delete all the entries in the population group options table? If all the entries have a date in the past? If so, what happens if this try statement fails after dropping all the records (e.g., fails on line 112), but doesn't replace the table with any records? Then we will have an empty table, right?

Reply:
yeah you're right... i can move the dropping old records to only happen if we've inserted new ones - good catch!